test/integration/pending.spec.js

```diff
@@ -59,6 +61,42 @@ describe('pending', function () {
             return;
           }
           assert.equal(res.stats.pending, 2);
+          assert.equal(res.stats.passes, 2);
+          assert.equal(res.stats.failures, 0);
+          assert.equal(res.code, 0);
+          done();
+        });
+      });
+      it('should run before and after hooks', function (done) {
+        runMocha('pending/skip-sync-before-hooks.fixture.js', args, function (err, res) {
```

Comment:
You *should* be able to get the titles out of `run` / `runJSON`; it's easier to use than inspecting the standard output.

Reply:
@boneskull The issue here is that I'm verifying hooks, not `describe` or `it` blocks. The log statements are used to verify that each hook executes / doesn't execute.

This pattern is based off [another test in the suite](https://github.com/mochajs/mocha/blob/master/test/integration/hooks.spec.js).